app/views/corporate_information_pages/show_worldwide_organisation.html.erb

```diff
@@ -8,7 +8,9 @@
     <% if headers.any? %>
       <nav>
         <div class="content">
-          <h1><%= t('worldwide_organisation.headings.contents', default: 'Contents') %></h1>
+          <h1 <%= "lang=#{t_fallback('worldwide_organisation.headings.contents')}" if t_fallback('worldwide_organisation.headings.contents') %>>
```

Comment:
I think the `t_lang` helper does what you need it to here. If it doesn't, I'd recommend putting `t_fallback('worldwide_organisation.headings.contents')` into a variable for brevity.